src/System.Net.Requests/tests/HttpWebRequestHeaderTest.cs

```diff
@@ -38,7 +38,8 @@ public void Ctor_VerifyHttpRequestDefaults(Uri remoteServer)
         }
 
         [OuterLoop]
-        [Theory, MemberData(nameof(EchoServers))]
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotFedoraOrRedHatOrCentos))]
+        [MemberData(nameof(EchoServers))]
```

Comment:
Please do so - maybe as commented "Disabled test attribute" - so that it is easy to search for ...